pkgs/tools/inputmethods/netevent/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, docutils }:
+
+stdenv.mkDerivation rec {
+  pname = "netevent";
+  version = "20201018";
+
+  src = fetchFromGitHub {
+    owner = "Blub";
+    repo = "netevent";
+    rev = "ddd330f0dc956a95a111c58ad10546071058e4c1";
+    sha256 = "0myk91pmim0m51h4b8hplkbxvns0icvfmv0401r0hw8md828nh5c";
+  };
+
+  buildInputs = [ docutils ];
+
+  outputs = [ "out" "doc" "man" ];
+
+  RST2MAN = "rst2man";
+
+  configurePhase = ''
+    ./configure
```

Comment:
Why override the configurePhase - it already runs ./configure. At the very least you should set --prefix=$out and remove the overridden installPhase below

Reply:
The ./configure is not supporting classical configure options, which means not overriding the configurePhase results in errors.